src/App.js

```diff
@@ -1,24 +1,30 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { NavLink, Route } from 'react-router-dom';
+import AddItem from './components/AddItem';
+import List from './components/List';
+
+// any css was added in index.css
```

Comment:
Since we have the import above, I think we can get rid of this comment. Great to note though, thank you! In general, comments are best when they focus on the why of a decision. So, for example, if I look at the code and can't quite understand something written in code, I like to see a comment explaining why it's written in a certian way.